addons/switch_focus/switch_focus.lua

```diff
@@ -0,0 +1,150 @@
+_addon = {}
+_addon.name = 'switch_focus'
+_addon.version = '1.0.0'
+_addon.author = 'WindowerDevTeam'
+_addon.commands = {'swf', 'switch', 'switch_focus'}
+
+require('tables')
+require('strings')
+local box_list = require('box_list')
+
+local back_name
+local switch = {cmd = {}}
+
+function switch.command(cmd, ...)
+    local args = T{...}:map(string.lower) or {}
+    cmd = cmd and cmd:lower() or 'help'
+    if switch.cmd[cmd] then
```

Comment:
Since you're already assigning a new variable before, may as well do:
```lua
    local command = switch.cmd[cmd and cmd:lower() or 'help']
```

Then you can just check that and save a few table lookups, and you wouldn't be overwriting a local variable. If you're feeling cheeky, you could also add `or functions.empty` after it, then you don't need the `if` anymore, and can just call it directly.